### ISSUE TITLE
Lemonadep will not flash -SPL

### ISSUE THREAD

**Issue by coffseducation** [lineageos4microg/docker-lineage-cicd#316]
I followed the instructions for flashing LineageOS for Microg from the Lineage website - using the LATEST vendor image and dbto.

Flashing the microg rom however will not work.
It says ":denying ota because it is a spl downgrade"

Will flashing the earlier version, (first build of LOS19 ) filess (boot.img, dbto, vendor and ?) downgrade the SPL version?

How can I flash the rom?

Thanks!

**Comment by SeanMcMillan** [comment]
I'm having the same problem on a Lemonade device.

If I'm reading the runes right on my investigation, it looks like the builds failed last month for some reason, which means version skew between Lineage+MicroG and upstream Lineage, which means when you grab dtbo+vendor_boot from upstream, you are getting a newer version than the ROM supports. Unfortunately, the older versions of those files have been purged, so you can't just go back to the June versions.

**Comment by coffseducation** [comment]
> I'm having the same problem on a Lemonade device.
> 
> If I'm reading the runes right on my investigation, it looks like the builds failed last month for some reason, which means version skew between Lineage+MicroG and upstream Lineage, which means when you grab dtbo+vendor_boot from upstream, you are getting a newer version than the ROM supports. Unfortunately, the older versions of those files have been purged, so you can't just go back to the June versions.

Yes, you are correct, you need the matching dbto and vendor files. Indeed these no longer appear to available prior to July.
Let me know if you want me to throw them up online somewhere?

**Comment by coffseducation** [comment]
> I'm having the same problem on a Lemonade device.
> 
> If I'm reading the runes right on my investigation, it looks like the builds failed last month for some reason, which means version skew between Lineage+MicroG and upstream Lineage, which means when you grab dtbo+vendor_boot from upstream, you are getting a newer version than the ROM supports. Unfortunately, the older versions of those files have been purged, so you can't just go back to the June versions.

Oh, sorry! You have Lemonade - not lemonadep (Pro). I do not have those files.

**Comment by SeanMcMillan** [comment]
You're too kind. I just went with Lineage + Google services and I'll reflash once the builds are working again.

**Comment by petefoth** [comment]
August `lemonadep` builds available https://download.lineage.microg.org/lemonadep/

@SeanMcMillan Please can you see if you can flash this build?
